xbmc/guilib/GUIFont.cpp

```diff
@@ -17,19 +17,21 @@
 
 #define ROUND(x) (float)(MathUtils::round_int(x))
 
-CScrollInfo::CScrollInfo(unsigned int wait /* = 50 */, float pos /* = 0 */,
-  int speed /* = defaultSpeed */, const std::string &scrollSuffix /* = " | " */)
+CScrollInfo::CScrollInfo(unsigned int wait /* = 50 */,
+                         float pos /* = 0 */,
+                         int speed /* = defaultSpeed */,
+                         const std::string& scrollSuffix /* = " | " */)
 {
-    initialWait = wait;
-    initialPos = pos;
-    SetSpeed(speed ? speed : defaultSpeed);
-    std::wstring wsuffix;
-    g_charsetConverter.utf8ToW(scrollSuffix, wsuffix);
-    suffix.clear();
-    suffix.reserve(wsuffix.size());
-    for (vecText::size_type i = 0; i < wsuffix.size(); i++)
-      suffix.push_back(wsuffix[i]);
-    Reset();
+  initialWait = wait;
+  initialPos = pos;
+  SetSpeed(speed ? speed : defaultSpeed);
+  std::wstring wsuffix;
+  g_charsetConverter.utf8ToW(scrollSuffix, wsuffix);
+  suffix.clear();
+  suffix.reserve(wsuffix.size());
+  for (vecText::size_type i = 0; i < wsuffix.size(); i++)
```

Comment:
This stuff can probably be replaced with something like:
```
m_suffix = vecText(wsuffix.begin(), wsuffix.end());
```

Reply:
Have added it and seems to work.